sources/packages/backend/workflow/BPMN/camunda-8/load-assessment-consolidated-data.bpmn

```diff
@@ -99,6 +99,7 @@
           <zeebe:header key="studentDataStudentParentNetAssests" value="$.data.studentParentNetAssests" />
           <zeebe:header key="studentDataStudentParentNetContribution" value="$.data.studentParentNetContribution" />
           <zeebe:header key="studentDataPartnerTotalIncomeAssistance" value="$.data.partnertotalincomeAssistance" />
+          <zeebe:header key="studentDataPartTimeAwardTypesToBeConsidered" value="$.data.partTimeAwardTypes" />
```

Comment:
I believe that we try to follow the patter where the `studentData` is juts added as a prefix without changing the variable name itself. Can we keep this, please?

It should be either
- studentDataPartTimeAwardTypesToBeConsidered
- partTimeAwardTypesToBeConsidered
or
- studentDataPartTimeAwardTypes
- partTimeAwardTypes